### ISSUE TITLE
Convert VM tests to state tests format (blockchain test format as well?)

### ISSUE THREAD

**Issue by jwasinger** [ethereum/testeth#33]
I think it would be useful to have a feature in testeth that converts tests from VM tests format to state tests (and/or blockchain tests) format during filling.

This makes it easier from the client perspective (not having to account for multiple test formats.

Thoughts @pirapira @winsvega ?  I could try and implement this (it would be useful for ewasm)

**Comment by pirapira** [comment]
Yes, that's a good idea.  Until `retesteth` gets the same feature, we can keep using `testeth` for that purpose.

@winsvega, have you translated VM tests into some other formats somehow automatically?

**Comment by winsvega** [comment]
no. in theory vmtests cases should be in state tests. 

I am up for a general test format. and it looks like its going to be blockchain tests.
